node-wasm/js/index.js

```diff
@@ -0,0 +1,11 @@
+import init, { NodeClient } from "lumina-node-wasm"
```

Comment:
please also add `index.d.ts` file with such content (I hope it's correct way to do it, but not 100% sure :p) and add it to list of files in `package.json`
```ts
/**
* Spawn a worker running lumina node and get the `NodeClient` connected to it.
*/
export function spawnNode(): Promise<NodeClient>;
export * from "lumina-node-wasm";
export default init;
```